### DISCUSSION TITLE
Update the Angular version to @12.X.X

### DISCUSSION THREAD

**Discussion by rajeevsathish** [Sunbird-Ed/Community#53] [General Discussion]
Hi,
Sunbird ED is planning to update the angular version from 11 to 12 by End of August 2022 
so request all dependent repos to update the angular version to 12. Please acknowledge or please update the plan for your repos ASAP.
Please prioritize this activity.
Below are the Repos and the team owning the same

Repo | Owned By
-- | --
SunbirdED-portal | SunbirdED
@project-sunbird/chatbot-client-v8 | SunbirdED
@project-sunbird/common-consumption-v9 | SunbirdED
@project-sunbird/common-form-elements-v9 | SunbirdED
@project-sunbird/discussions-ui-v8 |  
@project-sunbird/sunbird-epub-player-v9 | Player Team
@project-sunbird/sb-dashlet-v9 | SunbirdED
@project-sunbird/sunbird-collection-editor-v9 | Knowledge
@project-sunbird/sunbird-pdf-player-v9 | Player Team
@project-sunbird/sunbird-question-editor | Knowledge
@project-sunbird/sunbird-quml-player-v9 | Player Team
@project-sunbird/sunbird-video-player-v9 | Player Team
@project-sunbird/web-extensions | SunbirdED
@samagra-x/uci-console | Samagra
@shikshalokam/sl-questionnaire | ShikshaLokam
@shikshalokam/sl-reports-library | ShikshaLokam
@swimlane/ngx-datatable |  
common-form-elements-v9 | SunbirdED
common-form-elements-web-v9 | SunbirdED
sb-notification | SunbirdED
sb-svg2pdf | Player Team

@sharathkashyap @vinukumar-vs @Krishnaj20 @pallakartheekreddy @rhwarrier @reshmi-nair @anandp504 @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @SanthoshVasabhaktula @vrayulu @ashokreddy1208 @gandham-santhosh @kameshbhr @mmanimegalai @madhucr @alok-os

**Comment by rhwarrier** [comment]
@vinukumar-vs should we consider this for 5-1 Sprint 1 in that case ?

**Comment by vinukumar-vs** [comment]
Yes.

Vinu Kumar V S

Tech Architect

Mobile: +91 - 9964300623

------------------------------

Tarento Technologies, India

Explore@ www.tarento.com

<https://www.tarento.com/>

Please note that this message may contain privileged and confidential
information. This communication and any attachments are solely intended for
the person(s) or organization(s) to whom it is addressed. If you are not
the intended recipient, you are hereby notified that any unauthorized use,
copying or disclosure of this communication and its attachments is strictly
prohibited and may be unlawful. If you have received this communication
without being the intended recipient please discard the email without any
usage and/or distribution of the information and inform us.

**Comment by vaibhavbhuva** [comment]
@rajeevsathish Who is owning this package @project-sunbird/sb-styles? Because we are getting warning from this package while migrating the collection editor angular version from 11 to 12.

Please find the error logs below:

```
WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($size, 16px)

More info and automated migrator: https://sass-lang.com/d/slash-div

  ╷
3 │     $remSize: $size / 16px;
  │               ^^^^^^^^^^^^
  ╵
    node_modules/@project-sunbird/sb-styles/assets/mixins/_functions.scss 3:12  calculateRem()
    node_modules/@project-sunbird/sb-styles/assets/_icons.scss 98:10            @import
    node_modules/@project-sunbird/sb-styles/assets/_styles.scss 5:9             root stylesheet
```


cc: @vinukumar-vs  @pallakartheekreddy @sharathkashyap
